keyboards/converter/palm_usb/readme.md

```diff
@@ -23,9 +23,10 @@ qmk because  the Arduino softserial library uses different pins from QMK.
 
 I've wired the pro micro  hardware as follows. 
 
-Label| TX0,RX1,GND,GND,2  ,3  ,4  ,5  ,6  ,7
-Palm |    ,   , * ,GND,VCC,RX ,NC ,RTS,nc ,DCD
-MCU  |                ,D1 ,D0 ,   ,C6 ,   ,E6
+| Label | TX0 | RX1 | GND | GND | 2 | 3 | 4 | 5 | 6 | 7 |
```

Comment:
It would be nice if you could also align the columns here :)

Reply:
I thought they were. :)
Maybe I'm just missing something here. I took the original commit where I could see the intended formatting and applied the tabular format so it doesn't all run together.
I just built the pure usb version the other day and realized I could have a QMK version but was a bit confused by the connection info. When I looked at the commit itself it was pretty clear what was intended with the three rows. I'll happily clean it up more if something is lacking though.

Reply:
@fauxpark while it doesn't look so here, the rich view does show that it's formatted into a table.  So it should be good.

Reply:
I mean to say it looks fine rendered, but the Markdown itself is not aligned.

Reply:
I follow now. Since the white space in the source won't change the final rendering is what you meant. I've updated the code accordingly.